content/concepts/tokens/index.md

```diff
@@ -0,0 +1,53 @@
+---
+html: tk-index.html
```

Comment:
Similar with NFTs, this would reduce migration work and break fewer links if it maintained the `tokens.html` output filename.